x/ibc/04-channel/types/channel.go

```diff
@@ -13,9 +13,6 @@ var (
 	_ exported.CounterpartyI = (*Counterparty)(nil)
 )
 
-// DefaultChannelVersion defines the default channel version used during handshake.
-const DefaultChannelVersion = "1.0.0"
```

Comment:
there is no "channel version" only application versions for the associated channel.